makeflow/src/makeflow.c

```diff
@@ -1669,6 +1680,43 @@ int main(int argc, char *argv[])
 		makeflow_wrapper_docker_init(wrapper, container_image, image_tar);
 	}
 
+	if (batch_queue_type == BATCH_QUEUE_TYPE_MESOS) {
+		pid_t mesos_PID;
+		mesos_PID = fork();			
+		char *mesos_cwd;
+		mesos_cwd = path_getcwd();
+
+		char *mesos_py = "/afs/nd.edu/user37/ccl/software/external/mesos-0.26.0/amd64_linux26/lib/python2.6/site-packages";
+		char *cctools_path = getenv("CCTOOLS");
```

Comment:
Use `readlink("/proc/self/exe", buf, buf_size)` to get the path of `makeflow` executable, instead of using `CCTOOLS`